city_scrapers/spiders/chi_ssa_16.py

```diff
@@ -0,0 +1,105 @@
+    timezone = "America/Chicago"
+    start_urls = ["https://greektownchicago.org/about/ssa-16/"]
+
+    def parse(self, response):
+        # Meeting entries are contained in a list item without any specific class or id designation.
+        # Because of this, I filtered the present list items on the page with the ", 20" string
+        # (every meeting li has this in the present text from the year listed),
+        # which indicates the meeting occurred or is scheduled to occur
+        # and therefore contains relevant information to document.
+
+        for item in response.xpath('//li[contains(text(), ", 20")]'):
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description=self._parse_description(item),
+                classification=self._parse_classification(item),
+                start=self._parse_start(item),
+                end=self._parse_end(item),
+                all_day=self._parse_all_day(item),
+                time_notes=self._parse_time_notes(item),
+                location=self._parse_location(item),
+                links=self._parse_links(item),
+                source=self._parse_source(response),
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_title(self, item):
+        # There was no variation in types of meetings, so this was applicable for all.
+        return "Greektown Special Service Area Tax Commission #16 Public Meeting"
+
+    def _parse_description(self, item):
+        # This extracts the useful text provide to describe the services provided by the SSA.
+        return " ".join(
+            item.xpath('//span[text()="Improving the Community"]/parent::h4/parent::div/p/text()'
+                       ).extract()
+        )
+
+    def _parse_classification(self, item):
+        return COMMISSION
+
+    def _parse_start(self, item):
+        # Extracted all alphanumeric characters from the text of each meeting,
+        # which contained the dates of the meeting, then split into list containing M, D, Y
+        date = re.sub(r"[^a-zA-Z0-9]+", " ", item.xpath("text()").get()).split()
+
+        # Extracted just alphabetical characters from month, then used built-in strptime function
+        # in datetime to parse numeric month from string
+        month = re.sub(r"[^a-zA-Z]+", " ", date[0])
+        month = datetime.datetime.strptime(month, "%B").month
+
+        # Extracted numbers from day and year entries of date list to remove any errant characters
+        # (i.e. one entry said January 28th where all others read January 28).
+        day = int(re.sub(r"[^0-9]+", " ", date[1]))
+        year = int(re.sub(r"[^0-9]+", " ", date[2]))
+
+        # According to accompying information, all meetings in 2016 occurred at 3:00 PM
+        # All other years occurred at 2:00 PM.
+        if (year == 2016):
```

Comment:
If possible it would be good to parse this from row rather than hard coding it since each row of a year description and meeting list is in a separate `.gdc_row` div